PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/creation/PasswordUtils.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azure.toolkit.intellij.mysql.creation;
+
+import com.microsoft.azure.toolkit.intellij.common.AzurePasswordFieldInput;
+import com.microsoft.azure.toolkit.lib.common.form.AzureFormInput;
+import com.microsoft.azure.toolkit.lib.common.form.AzureValidationInfo;
+import org.apache.commons.lang3.CharUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.swing.*;
+
+public class PasswordUtils {
+
+    protected static AzurePasswordFieldInput generatePasswordFieldInput(JPasswordField passwordField, JTextField adminUsernameTextField) {
+        return new AzurePasswordFieldInput(passwordField) {
+            @Override
+            public AzureValidationInfo doValidate() {
```

Comment:
you could also create a password/confirmpassword input with different `com.microsoft.azure.toolkit.lib.common.form.Validatable.Validator` instances,

Reply:
seems it's not a good idea for my case.
if define validator as a seperate class, it needs more  two classes.
if use function method, I have two classes without extend-relationship need to use it.
util static method is more fit it.

Reply:
the final code would something like, 
```
this.passwordInput = new AzurePasswordFieldInput(originalPassworldInput);
this.passwordInput.setValidator(ValidationUtils.validatePassword);
...
this.confirmInput = new AzurePasswordFieldInput(originalConfirmInput);
this.confirmInput.setValidator(ValidationUtils.validateConfirmPassword);
```

Reply:
will consider to refactor it in my next PR.